### PR TITLE
Feat/LIVE-2652 - LLM - Asset centric wallet - Improvement and factoring of actions buttons management

### PR DESCRIPTION
**Pull request by nparigi-ledger**
### 📝 Description

- Use ui/icons for Tron actions icons
- Set return type for all getActions functions for families
- Change object structure of ActionButtonEvent type
- Add "FabButtonBarProvider" component : a provider that provides a list of props for action buttons from an action list, managing the navigation parameters, onPress functions and modals of these button bars
- If an asset have active accounts, but the balance is empty, then a click on a disable action will open a prompt to receive/buy

### ❓ Context

- **Impacted projects**: `` live-mobile
- **Linked resource(s)**: `` https://ledgerhq.atlassian.net/jira/software/c/projects/LIVE/boards/463?modal=detail&selectedIssue=LIVE-3587

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/FabActions/actionsList/account/index.tsx [hunk 1]
```diff
@@ -19,93 +15,36 @@ type FabAccountActionsProps = {
...
-          parentId,
-        },
-      });
-    },
-    [account, parentAccount, navigation],
-  );
-
-  const onPress = useCallback(
-    (data: ActionButton) => {
-      track("button_clicked", {
```
Comment (ThomasLaforge) [inline apps/ledger-live-mobile/src/components/FabActions/actionsList/account/index.tsx:51]: I don't see any event of this format on your rewrite

File: apps/ledger-live-mobile/src/components/FabActions/actionsList/account/index.tsx [hunk 2]
```diff
@@ -19,93 +15,36 @@ type FabAccountActionsProps = {
...
-    .map(action => ({
-      Icon: action.Icon,
-      children: action.label,
-      onPress: () => onPress(action),
-      disabled: action.disabled,
-      onPressWhenDisabled: action.modalOnDisabledClick
-        ? () => onPressWhenDisabled(action)
-        : undefined,
-    }))
-    .sort(a => (a.disabled ? 0 : -1));
```
Comment (ThomasLaforge) [inline apps/ledger-live-mobile/src/components/FabActions/actionsList/account/index.tsx:79]: I no more see it on your modifications. Is it still sorted by status ?

**Review by mlegall-ledger — COMMENTED** [review]
Great work !

**Review by ThomasLaforge — APPROVED** [review]
Thanks !

**Review by ThomasLaforge — APPROVED** [review]
Just check with @ahadwan-ledger if it's of for ids in lowercase

**Review by cgrellard-ledger — APPROVED** [review]

**Pull request merged**